src/utils/theme.js

```diff
@@ -0,0 +1,500 @@
+        },
+      },
+      MuiPickersClock: {
+        pin: {
+          backgroundColor: THEME.SECONDARY,
+        },
+      },
+      MuiPickersClockPointer: {
+        pointer: {
+          backgroundColor: THEME.SECONDARY,
+        },
+        thumb: {
+          borderColor: THEME.SECONDARY,
+          backgroundColor: THEME.PRIMARY_TEXT_DARK,
+        },
+        noPoint: {
+          backgroundColor: THEME.SECONDARY,
+        },
+      },
+      MuiPickersClockNumber: {
+        clockNumberSelected: {
+          color: '#fff',
+        },
+      },
+      MuiPickerDTTabs: {
+        tabs: {
+          color: '#fff',
+          backgroundColor: THEME.SECONDARY,
+          '& .MuiTabs-indicator': {
+            backgroundColor: isDarkTheme ? '#fff' : '#000',
+          },
+        },
+      },
+      MuiListItem: {
+        root: {
+          userSelect: 'text',
+        },
+      },
+      MuiChip: {
+        label: {
+          userSelect: 'text',
+        },
+        root: {
+          userSelect: 'text',
+        },
+      },
+      MuiPickersCalendarHeader: {
+        iconButton: {
+          backgroundColor: 'transparent',
+          '& span': {
+            backgroundColor: 'transparent',
+          },
+        },
+      },
+      MuiPickersDay: {
+        daySelected: {
+          backgroundColor: THEME.SECONDARY,
+          color: THEME.PRIMARY_TEXT_DARK,
+          '&:hover': {
+            backgroundColor: THEME.SECONDARY,
+          },
+        },
+        current: {
+          color: isDarkTheme
+            ? THEME.PRIMARY_TEXT_DARK
+            : THEME.PRIMARY_TEXT_LIGHT,
+        },
+      },
+      MuiPickersModal: {
+        dialogAction: {
+          color: isDarkTheme
+            ? THEME.PRIMARY_TEXT_DARK
+            : THEME.PRIMARY_TEXT_LIGHT,
+          '&:hover': {
+            backgroundColor: isDarkTheme
+              ? THEME.TEN_PERCENT_WHITE
+              : THEME.TEN_PERCENT_BLACK,
+          },
+        },
+        withAdditionalAction: {
+          '& button': {
+            color: isDarkTheme
+              ? THEME.PRIMARY_TEXT_DARK
+              : THEME.PRIMARY_TEXT_LIGHT,
+            '&:hover': {
+              backgroundColor: isDarkTheme
+                ? THEME.TEN_PERCENT_WHITE
+                : THEME.TEN_PERCENT_BLACK,
+            },
+          },
+        },
+      },
+      MuiDrawer: {
+        paper: {
+          maxWidth: 500,
+        },
+      },
+      MuiExpansionPanelSummary: {
+        root: {
+          '&$focused': {
+            backgroundColor: THEME.TEN_PERCENT_WHITE,
+          },
+        },
+      },
+      MuiMenuItem: {
+        root: {
+          color: isDarkTheme
+            ? THEME.PRIMARY_TEXT_DARK
+            : THEME.PRIMARY_TEXT_LIGHT,
+        },
+      },
+    },
+  };
+};
+
+export const theme = createMuiTheme(createTheme(true));
+
+export const TASKCLUSTER_THEME = {
+  lightTheme: createMuiTheme(createTheme(false)),
+  darkTheme: theme,
+  styleguide: {

```

Comment:
We can remove everything in `styleguide`. That's used for Taskcluster's styleguide, not the main site.

Reply:
Applied changes!